### ISSUE TITLE
Java.registerClass failed for extended interface

### ISSUE THREAD

**Issue by gebing** [frida/frida-java#124]
Java.registerClass failed for implement extended interface when using function implement, such like below:

```
public interface Eatable {
    String getName();
    int getCalories(int grams);
}

public interface EatableEx extends Eatable {
  String getNameEx();
}
```

and javascript is like:

```
var EatableEx = Java.use('re.frida.EatableEx');
var BananaEx = Java.registerClass({
  name: 're.frida.BananaEx',
  implements: [EatableEx],
  methods: {
    getName: function () {
      return 'Banana';
    },
    getNameEx: function () {
      return 'BananaEx';
    },
    getCalories: function (grams) {
      return grams * 2;
    },
  }
});
```

And this will cause below exception:

```
java.lang.AbstractMethodError: abstract method "java.lang.String re.frida.Eatable.getName()"
        at re.frida.ClassCreationTest.extendInterfaceCanBeImplemented(ClassCreationTest.java:191)
        at java.lang.reflect.Method.invoke(Native Method)
```

**Comment by gebing** [comment]
I found it could be solved by adding base interface to implements, such like below:

```
var Eatable = Java.use('re.frida.Eatable');
var EatableEx = Java.use('re.frida.EatableEx');
var BananaEx = Java.registerClass({
  name: 're.frida.BananaEx',
  implements: [EatableEx, Eatable],
  methods: {
    getName: function () {
      return 'Banana';
    },
    getNameEx: function () {
      return 'BananaEx';
    },
    getCalories: function (grams) {
      return grams * 2;
    },
  }
})
```

So there is no need to modify the source code, and i will close this issue.

**Comment by oleavr** [comment]
Thanks for looking into this! I just addressed this in 7700fd9e47f1eeb52d5b35cc943e5c0e5cc661fa so there is no need for specifying base interfaces, as this is not very intuitive.
